poetry.lock

```diff
@@ -509,6 +523,17 @@ optional = false
 python-versions = ">=2.6, !=3.0.*, !=3.1.*, !=3.2.*"
 version = "2.4.6"
 
+[[package]]
+category = "main"
+description = "Persistent/Functional/Immutable data structures"
+name = "pyrsistent"
```

Comment:
Did this get committed by mistake, or is this one of marshmallow's dependencies?

Reply:
Yep, that was a mistake.